.github/workflows/pr.yaml

```diff
@@ -123,6 +123,10 @@ jobs:
           do
             sleep 3
           done
+      - name: Test delete
+        timeout-minutes: 1
+        run: |
+          kubectl delete konks,konkservices --all --all-namespaces
```

Comment:
We should check with helm that the operator actually completed the task.
```suggestion
          kubectl delete konks,konkservices --all --all-namespaces
          helm -n $KONK_NAMESPACE | grep -c runner-konk 
```

Reply:
might need to add `set -e` to get github-actions to fail on matches

Reply:
`set -e` is default in actions.

Reply:
Added support for this.